plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -384,7 +384,7 @@ private static void validateCertificates(KeyStore keyStore)
             String nodeId = entry.getKey();
             NodesResponse.Node node = entry.getValue();
 
-            if (node.getRoles().contains("data")) {
+            if (node.getRoles().contains("data") || node.getRoles().contains("data_hot") || node.getRoles().contains("data_content")) {
```

Comment:
Can we capture these roles in a `Set` and do `Set#contains` ?

Reply:
done